internal/pkg/skuba/util/retry.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Retry(attempts int, sleep time.Duration, fn func() error) error {
+	if err := fn(); err != nil {
+		if attempts--; attempts > 0 {
+			jitter := time.Duration(rand.Int63n(int64(sleep)))
+			sleep = sleep + jitter/2
+			time.Sleep(sleep)
+			return Retry(attempts, 2*sleep, fn)
+		}
+		return err
+	}
+	return nil
+}
```

Comment:
We should use https://github.com/kubernetes/kubernetes/blob/b141a990c39da1295c8be521a9dea3317d67082e/staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go#L290, or https://github.com/kubernetes/kubernetes/blob/b141a990c39da1295c8be521a9dea3317d67082e/staging/src/k8s.io/client-go/util/retry/util.go#L48 depending on the purpose.

Reply:
done!